java-management/set-up-of-management-agent/set-up-of-management-agent.md

```diff
@@ -56,86 +56,91 @@ This task is for the installation of Management Agent in an OCI Host(OCI Compute
 
   ![image of jms policy](images/save-jms-policy.png)
 
-**Download installation script**
+**Download agent installer software**
 
-1. To Download installation script, in the Oracle Cloud Console, open the navigation menu and click **Observability & Management**, and then click **Fleets** under **Java Management**.
+1. To Download agent installer software, in the Oracle Cloud Console, open the navigation menu and click **Observability & Management**, and then click **Fleets** under **Java Management**.
 
 ![image of navigate to jms-console](images/console-navigation-jms.png)
 
 2. Select the Fleet created in [Lab 3](?lab=setup-a-fleet).
-3. Click **configure managed instances**
+3. Click **Download Agent Installer**
 
-![image of clicking configuration](images/click-configuration.png)
+![image of clicking configuration](images/DownloadAgentInstaller.png)
```

Comment:
follow the same naming covention (dash-case). Applies to other file names as well

Reply:
Changed. Affected files: Agent-installer-close.png, Download-Agent-Installer.png, Select-Agent-Installer.png

Reply:
can you make it all lower case also?

Reply:
All affected files changed to lower case.

Reply:
![image](https://github.com/user-attachments/assets/0a3c0bcd-0c50-42ea-bcbe-49ee3fc673e9)
some file names are still ProperCase

Reply:
My bad. All affected files changed.